pyplot/baselineFits/.gitignore

```diff
@@ -0,0 +1,2 @@
+old/
```

Comment:
It is not a clean style to put an ignore here to ignore stuff only you have locally. Please remove and change workflow to not run these directly in the project source directories.